sympy/solvers/inequalities.py

```diff
@@ -982,3 +985,246 @@ def reduce_inequalities(inequalities, symbols=[]):
+
+            piv_cols = []
+            for j in range(A.cols):
+                if A[k, j] < 0:
+                    piv_cols.append(j)
+            if not piv_cols:
+                raise InfeasibleLinearProgrammingError('The constraint set is empty!')
+            rand.shuffle(piv_cols)
+            j0 = piv_cols[0]
+
+            ratio = B[k] / A[k, j0]
+            piv_rows = [(ratio, k)]
+            for i in range(A.rows):
+                if A[i, j0] > 0 and B[i] > 0:
+                    ratio = B[i] / A[i, j0]
+                    piv_rows.append((ratio, i))
+
+            piv_rows = sorted(piv_rows, key=lambda x: (x[0], x[1]))
+            piv_rows = [(ratio, i) for ratio, i in piv_rows if ratio == piv_rows[0][0]]
+            rand.shuffle(piv_rows)
+            _, i0 = piv_rows[0]
+
+            M = _pivot(M, i0, j0)
+            R[j0], S[i0] = S[i0], R[j0]
+
+def _to_standard_form(constraints, objective):
+    """ Converts a list of constraints and an objective function into the standard form for linear programming
+
+    Examples
+    ========
+
+    >>> from sympy.solvers.inequalities import _to_standard_form
+    >>> from sympy.abc import x, y, z
+    >>> from sympy import Eq
+    >>> r1 = y+2*z <= 3
+    >>> r2 = -x-3*z <= -2
+    >>> r3 = 2*x+y+7*z <= 5
+    >>> A, B, C = _to_standard_form([r1, r2, r3], x+y+5*z)
+    >>> A
+    Matrix([[1, 0, 2], [0, -1, -3], [1, 2, 7]])
+    >>> B
+    Matrix([[3], [-2], [5]])
+    >>> C
+    Matrix([[1, 1, 5]])
+    >>> r1 = Eq(x, y)
+    >>> A, B, C = _to_standard_form([Eq(x,3)],x*10) # x = 3 become x >= 3 and x <= 3
+    >>> A
+    Matrix([[1], [-1]])
+    >>> B
+    Matrix([[3], [-3]])
+    >>> C
+    Matrix([[10]])
+    """
+    A = [];
+    B = [];
+    free_sym = objective.free_symbols
+
+    for rel in constraints:
+        if not isinstance(rel, Relational):
+            raise TypeError(f"{rel} is not relational.")
+        if type(rel) in [Lt, Gt]:
+            raise TypeError("Strict inequalities are not allowed in linear programming.")
+        if type(rel) == Ne:
+            raise TypeError("'not equal to' is not allowed in linear programming.")
+        if not rel.rhs.is_constant():
+            raise ValueError("Right hand side of relation must be constant")
+
+        free_sym = free_sym | rel.free_symbols
+
+        if type(rel) == Le:
+            A.append(rel.lhs)
+            B.append(rel.rhs)
+        elif type(rel) == Ge:
+            A.append(-rel.lhs)
+            B.append(-rel.rhs)
+        elif type(rel) == Eq:
+            # x = 3 can be represented as x <= 3 and x >= 3
+            A.append(rel.lhs)
+            B.append(rel.rhs)
+            A.append(-rel.lhs)
+            B.append(-rel.rhs)
+        else:
+            raise TypeError(f"Unrecognized relational: {rel}")
+
+    A = Matrix([[row.coeff(sym) for sym in free_sym] for row in A])
+    B = Matrix(B)
+    C = Matrix([[objective.coeff(sym) for sym in free_sym]])
+
+    return A, B, C
```

Comment:
So the user doesn't have to guess the order of variables, it would be good to return the order used to construct the matrices. In this case it's not strictly necessary to order them as suggested above since the returned order would remove ambiguity.